packages/terra-functional-testing/lib/commands/run-axe.js

```diff
@@ -0,0 +1,37 @@
+/* global browser, axe */
+const injectAxe = require('./inject-axe');
+
+/**
+ * Executes axe on the browser.
+ * @param {Object} overrides - The axe options.
+ * @param {Array} overrides.rules - The rule overrides
+ */
+const runAxe = (overrides = {}) => {
+  const isAxeUnavailable = browser.execute(() => window.axe === undefined);
+
+  // Inject axe-core onto the page if it has not already been initialize.
+  if (isAxeUnavailable) {
+    injectAxe();
+  }
+
+  // Extract the axe options for the Terra service from the global browser object.
+  const [, options = {}] = browser.options.services.find(([service]) => (
+    typeof service === 'function' && service.name === 'TerraService'
+  ));
+
+  // Merge the global rules and overrides together.
+  const rules = {
+    ...options.axe && options.axe.rules && options.axe.rules.reduce((acc, rule) => ({ ...acc, [rule.id]: rule }), {}),
+    ...overrides.rules,
+  };
+
+  // eslint-disable-next-line prefer-arrow-callback, func-names
+  return browser.executeAsync(function (opts, done) {
```

Comment:
Should we be returning the value of the result?

Reply:
No, this must have been on of the non-passive changes from WDIO 4 to WDIO 6. Here's a compressed output of this return statement:

```
[0-0] { error: null,
  result:
   { inapplicable:
      [ [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object] ],
     incomplete: [ [Object] ],
     passes:
      [ [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object],
        [Object] ],
     testEngine: { name: 'axe-core', version: '3.5.4' },
     testEnvironment:
      { orientationAngle: 0,
        orientationType: 'landscape-primary',
        userAgent:
         'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.61 Safari/537.36',
        windowHeight: 884,
        windowWidth: 1050 },
     testRunner: { name: 'axe' },
     timestamp: '2020-06-17T15:22:26.237Z',
     toolOptions:
      { reporter: 'v1',
        restoreScroll: true,
        rules: {},
        runOnly: [Object] },
     url:
      'https://engineering.cerner.com/terra-ui/home/terra-ui/index',
     violations: [] } }
````